ingest-app/src/cmr/ingest/api/generic_documents.clj

```diff
@@ -230,7 +230,9 @@
              request-context :update :provider-object provider-id)
             (acl/verify-provider-context-permission
              request-context :read :provider-object provider-id))]
-    (api-core/delete-concept concept-type provider-id native-id request)))
+    (if (is-draft-concept? request)
+      (api-core/delete-draft concept-type provider-id native-id request)
```

Comment:
Does the `delete-draft` function in ingest/core leave a tombstone? If it does not we should update the doc-string in this function with something like `Deletes a generic document in elasticsearch and creates a tombstone in the database if the concept is not a draft concept.`

Reply:
updated the doc string.